os/include/tinyara/sched.h

```diff
@@ -80,25 +80,10 @@
 #include <tinyara/mm/shm.h>
 #include <tinyara/fs/fs.h>
 #include <tinyara/net/net.h>
+#include <tinyara/mpu.h>
```

Comment:
I think header mpu.h should be included under CONFIG_MPU_STACK_OVERFLOW_PROTECTION check

Reply:
I have created mpu.h to include generic definitions, it also include the usage for app_binary_separation, stack overflow.

Reply:
I have created mpu.h to include generic definitions, it also include the usage for app_binary_separation, stack overflow.